sdk/search/azure-search-documents/src/main/java/com/azure/search/documents/models/VectorizableTextQuery.java

```diff
@@ -55,6 +60,28 @@ public String getText() {
         return this.text;
     }
 
+    /**
+     * Get the queryRewrites property: Can be configured to let a generative model rewrite the query before sending it
+     * to be vectorized.
+     *
+     * @return the queryRewrites value.
+     */
+    public QueryRewritesType getQueryRewrites() {
+        return this.queryRewrites;
+    }
+
+    /**
+     * Set the queryRewrites property: Can be configured to let a generative model rewrite the query before sending it
+     * to be vectorized.
+     *
+     * @param queryRewrites the queryRewrites value to set.
+     * @return the VectorizableTextQuery object itself.
+     */
+    public VectorizableTextQuery setQueryRewrites(QueryRewritesType queryRewrites) {
```

Comment:
We'll want to switch this over to `QueryRewrites`, but this may force us to vendor `VectorizableTextQuery` or a significant customization.

Reply:
I'll go the route of customization.